CMakeLists.txt

```diff
@@ -95,9 +95,10 @@ option(IREE_ENABLE_NEW_INTEGRATION_TESTS "Enables new integration tests and disa
 # Derived flags based on primary options
 #-------------------------------------------------------------------------------
 
```

Comment:
Could you just move the IREE_ENABLE_EMITC option here and default it to the value of IREE_BUILD_COMPILER?

Reply:
That would be possible as well. However, I think in that case it wont be possible to build the compiler without emitc. If it is, we even could drop `IREE_ENABLE_EMITC` and replace its use by `IREE_BUILD_COMPILER`.